doc/rfcs/003-quickly-collaborate-with-coworkers-and-friends.md

```diff
@@ -0,0 +1,126 @@
+# Quickly collaborate with coworkers and friends
+
+## Status
+
+Proposed
+
+## Summary
+
+A host can share a URL for the portal, and guests can follow the URL to instantly join the portal. (Users will typically share these URLs via a third-party chat service like Slack, IRC, or similar.) Then, once you've collaborated with someone for the first time, you can establish future collaboration sessions with them again directly within Atom.
+
+## Motivation
+
+We hope to encourage more collaboration by reducing the barriers to entry. Specifically, we want to reduce the number of steps that exist between A) deciding that you *want* to collaborate and B) *actually* collaborating.
+
+Today, the transition from A to B involves the following steps:
+
+1. Host shares a portal
+2. Host copies portal ID to their clipboard
+3. Host switches from Atom to third-party communication tool (e.g., Slack, IRC)
+4. Host pastes portal ID into third-party communication tool
+5. Guest selects portal ID and copies it to their clipboard
+6. Guest switches from third-party communication tool to Atom
+7. Guest clicks "Join a portal" link or invokes "Join portal" command, and Atom automatically fetches the portal ID from the clipboard
+8. Guest clicks "Join"
+
+With the ability to join a portal via a URL, we can reduce the guest's set-up process from 4 steps to just 1 step:
+
+1. Host shares a portal
+2. Host copies portal URL to their clipboard
+3. Host switches from Atom to third-party communication tool (e.g., Slack, IRC)
+4. Host pastes portal URL into third-party communication tool
+5. Guest clicks portal URL, and the operating systems hands control to Atom, and Atom joins the portal
+
+With the ability to invite past collaborators to your portal from within Atom, we reduce the host's set-up process from 4 steps to 3 steps:
+
+1. Host shares a portal
+2. Host selects person in their list of past collaborators
+3. Host invites person to join the portal
+4. Guest accepts invitation to join portal, and Atom joins the portal
+
+And perhaps we can reduce the host's set-up process even further:
+
+1. Host selects a person in their list of past collaborators
+2. Host invites the selected person to a portal, and Atom automatically shares a portal for the host and invites the selected person
+3. Guest accepts invitation to join portal, and Atom joins the portal
+
+## Explanation
+
+### Portal URLs
+
+*TODO* Define URL format
```

Comment:
I know this still needs to be written, but it might be worth mentioning that we currently use `atom://teletype/portal/123/editor/345` to represent editor URIs in Atom's workspace.